reactos/drivers/usb/usbhub_new/usbhub.c

```diff
@@ -1124,6 +1128,11 @@ USBH_SyncGetHubDescriptor(IN PUSBHUB_FDO_EXTENSION HubExtension)
         NumberOfBytes = HubDescriptor->bDescriptorLength;
         ExFreePoolWithTag(HubDescriptor, USB_HUB_TAG);
 
+        if (Retry >= 5)
+        {
+            break;
```

Comment:
usbhub.c:1133: in this Retry >= 5 case, you're breaking out of the loop with HubDescriptor freed, but the code after the loop accesses HubDescriptor. Do you perhaps want a HubDescriptor=NULL/goto ErrorExit here instead?